rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/IRascalFileSystemServices.java

```diff
@@ -341,9 +341,9 @@ public boolean isRecursive() {
 
     public static class SourceLocation {
         private final String uri;
-        private final int[] offsetLength;
-        private final int[] beginLineColumn;
-        private final int[] endLineColumn;
+        private final int @Nullable[]  offsetLength;
```

Comment:
Learned something again. I did not know you could set annotations between the element type and the array indicator.